deviation_analysis.py

```diff
@@ -0,0 +1,52 @@
+# script to run to analyze the deviation data. Histograms, boxplots, and metrics are generated for each currency analyzed.
+import pandas as pd
+import matplotlib.pyplot as plt
+import json
+
+THRESHOLD=0.1
+
+with open('dev_amplitude.json', 'r') as file:
+    data = json.load(file)
+
+dfs = {currency: pd.DataFrame(records) for currency, records in data.items()}
+
+for currency, df in dfs.items():
+    original_count = df.shape[0]
+    
+    # Filter out rows where deviation is NaN or less than the threshold
+    df_cleaned = df.dropna(subset=['deviation'])
+    #df_cleaned = df_cleaned[df_cleaned['deviation'] > THRESHOLD]
```

Comment:
So for your analysis results v2 in the Notion page, we are not filtering out any values based on this `THRESHOLD`, correct?

Reply:
Correct, no filtering since it can be done in the python script and avoid re-running.